sdk/keyvault/platform-matrix.json

```diff
@@ -21,12 +21,6 @@
           "Pool": "azsdk-pool-mms-ubuntu-2004-general",
           "TestGoals": "surefire:test",
           "TestFromSource": true
-        },
-        "windows-2019_am_verify": {
```

Comment:
Any reason why this was removed?

Reply:
Running two HSM tests in parallel sometimes made it problematic for some scenarios, for example, you cannot run a restore operation while another one is in progress. The `FromSource` build already runs a `verify` step so I though it is would be redundant to have a separate `verify` run.